build.sbt

```diff
@@ -105,7 +105,7 @@ assemblyMergeStrategy in assembly := {
 // Settings from sbt-lighter plugin that will automate creating and submitting this job to EMR
 import sbtlighter._
 
-sparkEmrRelease := "emr-5.13.0"
+sparkEmrRelease := "emr-5.20.0"
```

Comment:
Bumps to spark 2.4.0 to match the version used by GT 2.2.0.
This is not likely to produce any issues, just house-keeping